src/pages/workspace/WorkspaceSidebar.js

```diff
@@ -65,14 +74,15 @@ const WorkspaceSidebar = ({translate, isSmallScreenWidth, policy}) => {
         },
     ];
 
-    useEffect(() => {
-        if (_.isEmpty(policy)) {
-            Growl.error(translate('workspace.error.growlMessageInvalidPolicy'), CONST.GROWL.DURATION_LONG);
-            Navigation.dismissModal();
-            create();
-            return null;
-        }
-    }, [policy]);
+    // After all the policies have loaded, we can know if the given policyID points to a nonexistant workspace
+    // @TODO: When free plan is out of beta and Permissions.canUseFreePlan() gets removed,
```

Comment:
```suggestion
    // When free plan is out of beta and Permissions.canUseFreePlan() gets removed,
```
(see other discussion... this will just be dealt with as a GH)

Reply:
Just got rid of it I think in the time you were doing the review!